src/System.Management.Automation/engine/lang/scriptblock.cs

```diff
@@ -647,16 +647,17 @@ internal ReadOnlyCollection<PSTypeName> OutputType
         /// <remarks>
```

Comment:
@daxian-dbw, your last commit had 2 failures in `PowerShell-CI-windows`
Language Primitive Tests.Convert ScriptBlock to delegate type
```powershell
Unable to find type [System.Func].
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Api\LanguagePrimitive.Tests.ps1: line 149
149:         $handlerReturnEnum = [System.Func[string, TestEnum]] { param([string]$str) "Music" }

```
Enter-PSHostProcess tests.By CustomPipeName.Can enter, exit, and re-enter using CustomPipeName
```powershell
Expected $true, because The script was able to re-enter another process and grab the pipe of 'n5jbsle2.r4a'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 170
170:                     Should -BeTrue -Because "The script was able to re-enter another process and grab the pipe of '$pipeName'."
```